packages/@okta/vuepress-site/docs/guides/build-provisioning-integration/test-scim-api/index.md

```diff
@@ -26,47 +26,77 @@ To get started using Runscope to test your SCIM API:
-The test results can be viewed in detail, but the test can't be edited or be re-run by people outside of your team.
-
-### Customize the imported Runscope test for your SCIM integration
-
-After importing the Okta SCIM test suite into Runscope, you need to configure the test for your SCIM integration:
-
-1. You should be looking at the **API Tests** screen in Runscope, if not, click the **Tests** tab at the top of the Runscope user interface.
+1. You should be looking at the **API Tests** window in Runscope, if not, click the **Tests** tab at the top of the Runscope user interface.
 1. You should see a test named **Okta SCIM X.X Spec Test** (where X.X corresponds to the SCIM version of the JSON file you uploaded).
 1. Click **Edit** on the test panel.
-1. The **Okta SCIM X.X SPEC Test** screen appears. Under the **Environment** section of your test, there is a collapsed section labelled **Test Settings** . Click the small arrow to expand this section.
+1. The **Okta SCIM X.X SPEC Test** window appears. Under the **Environment** section of your test, there is a collapsed section labelled **Test Settings** . Click the small arrow to expand this section.
 1. Select the **Initial Variables** tab.
+
+    ![Initial Variables tab](/img/oin/scim_test-init_variables.png "Showing the location of the Initial Variables tab")
 1. Click **Add Initial Variable** and add the following case-sensitive variables:
 
-    | Variable Name | Example Values              | Notes                                                                                                                                                                                     |
-    |:-------------------------------|:----------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-    | SCIMBaseURL                    | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
-    | auth                           | Bearer abcxyz1234567890     | Basic or OAuth authorization token                                                                                                                                                          |
+    | Variable Name | Example Values | Notes |
+    |:-|:-|:-|
+    | SCIMBaseURL | `https://example.com/scim/v2`  `https://example.com/scim/v1` | For example, if your SCIM integration is hosted on <https://example.com> and uses a prefix of /scim/v2 then the *SCIMBaseURL* value for your integration would be: `https://example.com/scim/v2`. |
+    | auth | Bearer abcxyz1234567890 | Basic or OAuth authorization token. |
```

Comment:
I'm not sure what this should be, but AFAIK `Basic authentication` doesn't use tokens, so "Basic authorization token" is a misnomer. In contrast to Basic Auth, OAuth tokens are Bearer tokens.

Reply:
Would "OAuth Bearer token, or Basic authentication code" fix the description?

Reply:
Changed to 

> OAuth 2.0 Bearer token or Basic authentication code.